src/main/java/seedu/mtracker/LogHelper.java

```diff
@@ -0,0 +1,50 @@
+package seedu.mtracker;
+
+import java.io.IOException;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+public class LogHelper {
+
+    public static final String LOG_FILE_NAME = "logger.log";
+
+    public static final String LOG_FILE_ERROR = "Log file not working!";
+    public static final String LOG_INVALID_NAME = "User tried to enter an invalid name here";
+    public static final String LOG_INVALID_PRICE = "User tried to enter an invalid price here";
+    public static final String LOG_INVALID_SENTIMENT = "User tried to enter an invalid sentiment here";
+
+    private static LogHelper logHelper;
+    private final Logger logger;
+
+    private LogHelper() {
+        logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+        setupLogger();
+    }
+
+    public static LogHelper getInstance() {
+        if (logHelper == null) {
+            logHelper = new LogHelper();
+        }
+        return logHelper;
+    }
+
+
+    public Logger getLogger() {
+        return logger;
+    }
+
+    public void setupLogger() {

```

Comment:
should we set up a console handler as well?

Reply:
From the video it seems like the consolehandler is meant to display messages onto the console, as for now I don't really know what scenario or situation is severe enough that should be displayed to the user beyond our current exceptions